media-sound/flacon/flacon-2.1.0.ebuild

```diff
@@ -0,0 +1,101 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+# Ignore rudimentary et, uz@Latn, zh_TW translation(s).
+PLOCALES="cs cs_CZ de es es_MX fr gl hu it ja_JP lt nb nl pl pl_PL pt_BR pt_PT ro_RO ru sr sr@latin tr uk zh_CN"
+
+inherit cmake-utils eutils gnome2-utils l10n virtualx xdg-utils
+
+DESCRIPTION="Extracts audio tracks from an audio CD image to separate tracks"
+HOMEPAGE="https://flacon.github.io/"
+SRC_URI="https://github.com/${PN}/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="LGPL-2.1+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="qt5 test"
+
+COMMON_DEPEND="
+	dev-libs/uchardet
+	!qt5? (
+		dev-qt/qtcore:4
+		dev-qt/qtgui:4
+	)
```

Comment:
While true, just because there are still some unfortunate kdelibs4-based KDE Applications around, that does not mean everything else gets to stall migration. In those rare cases where a KDE Application provides both (iirc e.g. umbrello, kmix), the Qt4 config is ignored. If Qt5 works well, there is no reason to keep Qt4 support. But at least this one does without REQUIRED_USE.